client/src/components/header-navigation/header-navigation.html

```diff
@@ -0,0 +1,8 @@
+<header class="header">
+    <div class="header__logo">Football stats</div>
+    <nav class="header__navigation">
+        <a class="header__link" href="#league" data-bind="css: {'header__link--selected': url() === '#league'}">league table</a>
```

Comment:
These URLs should be configurable and get here from params. Moreover, it also should be a pureComputed value, possibly generated by a fabric method.